lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -359,6 +359,10 @@ def test_cross_Qt_imports():
                     # are doing something that we do not expect to work
                     if ex.returncode == -11:
                         continue
+                    # We got the abort signal which is likely because the Qt5 /
+                    # Qt6 cross import is unhappy, carry on.
+                    elif ex.returncode == -6:
```

Comment:
perhaps `== -signal.SIGABRT` here (and likewise `-signal.SIGSEGV` just above)?

Reply:
done.